dspace-server-webapp/src/test/java/org/dspace/app/rest/csv/CSVMetadataImportReferenceTest.java

```diff
@@ -0,0 +1,399 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.csv;
+
+import static junit.framework.TestCase.assertEquals;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStreamWriter;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.UUID;
+
+import org.dspace.app.rest.builder.CollectionBuilder;
+import org.dspace.app.rest.builder.CommunityBuilder;
+import org.dspace.app.rest.builder.ItemBuilder;
+import org.dspace.app.rest.test.AbstractEntityIntegrationTest;
+import org.dspace.content.Collection;
+import org.dspace.content.Item;
+import org.dspace.content.MetadataField;
+import org.dspace.content.MetadataValue;
+import org.dspace.content.Relationship;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.ItemService;
+import org.dspace.content.service.MetadataFieldService;
+import org.dspace.content.service.MetadataValueService;
+import org.dspace.content.service.RelationshipService;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Created by: Andrew Wood
+ * Date: 26 Jul 2019
+ */
+public class CSVMetadataImportReferenceTest extends AbstractEntityIntegrationTest {
+
+    //Common collection to utilize for test
+    private Collection col1;
+
+    @Autowired
+    private RelationshipService relationshipService;
+
+    @Autowired
+    private ItemService itemService;
+
+    /**
+     * Setup testing enviorment
+     */
+    @Before
+    public void setup() {
+        context.turnOffAuthorisationSystem();
```

Comment:
I just realized we are turning *off* the authorizationSystem here without ever turning it back on.  This could have unintended consequences in these tests.  We need to turn it back *on* after creating the content in this `setup()` method.

Reply:
> @AndrewZWood : I gave this another code review today. It's looking mostly great, but I noticed three minor things that would be good to fix prior to merging. After these three things are fixed, I'm a +1.

Addressed all comments you brought up here: https://github.com/DSpace/DSpace/pull/2471/commits/591df53e7a19e1227e4b0d9db54f9a5bd69db27f#diff-5ce04a4fb4f137714321ddf616041328